misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/expression/NewExpressionEvaluator.java

```diff
@@ -122,17 +152,16 @@ private Value invokeObjectInitMethod(String className) throws EvaluationExceptio
                 CREATE_OBJECT_VALUE_METHOD, argTypeNames);
 
         List<Value> argValues = new ArrayList<>();
-        argValues.add(EvaluationUtils.getAsJString(context, context.getPackageOrg().orElse("")));
-        argValues.add(EvaluationUtils.getAsJString(context, context.getPackageName().orElse("")));
-        argValues.add(EvaluationUtils.getAsJString(context, context.getPackageMajorVersion().orElse("")));
-        argValues.add(EvaluationUtils.getAsJString(context, className));
+        argValues.add(getAsJString(context, moduleId.orgName()));
+        argValues.add(getAsJString(context, moduleId.moduleName()));
+        argValues.add(getAsJString(context, moduleId.version().split("\\.")[0]));
```

Comment:
Refer this [comment](https://github.com/ballerina-platform/ballerina-lang/pull/32621/files#r704925279)